### DISCUSSION TITLE
Is anyone else getting a "Declaration of case-insensitive constants is deprecated" error?

### DISCUSSION THREAD

**Discussion by fyrfli** [YOURLS/YOURLS#3252] [Q&A] [answered]
Hello all! 

Just wondering whether anyone else is getting this error at the top of their installation:

`define(): Declaration of case-insensitive constants is deprecated in config.php on line 41.`

I am running php8.1.2 and wondering if that is the issue. I sometimes cut myself on bleeding edge stuff so ... anyone have a bandaid?

😁

**Comment by ozh** [comment]
Nope. For the record this can happen since [7.3](https://php.watch/versions/7.3/constant-case-insensitivity-deprecated). What constant are you defining on line 41 ?

**Reply by fyrfli** [reply]
Interesting.

It's the define( 'YOURLS_SITE' ... line. I have the Allow Aliases plugin so there are two URLs on that line. 

`define( 'YOURLS_SITE', 'http://mysite.org','http://myothersite.org' );`

**Reply by ozh** [reply]
`define` doesn't work that way. It's `define ( 'THE CONSTANT NAME', "its value" );` and that's it. You cannot pass multiple values or whatever you're trying to do here, which results in `YOURLS_SITE` only being `http://mysite.org`  anyway. And the plugin you're referring to doesn't works that way either.

**Reply by fyrfli** [reply]
I see now. The plugin does not need anything to be changed in the config file. It works fine just activated in the admin panel. Wonder where I got that idea from? Anyway, again .. .thanks for your time.

**Comment by fyrfli** [comment]
Huh ... looks like I have to go re-read all the docs. Thanks for your time, though. Appreciated.

**Comment by fyrfli** [comment, marked as answer]
> I see now. The plugin does not need anything to be changed in the config file. It works fine just activated in the admin panel. Wonder where I got that idea from? Anyway, again .. .thanks for your time.

As I noted in the above thread ... no need to alter the config file for the Allow Aliases plugin to work.
